crates/common_enums/src/enums.rs

```diff
@@ -1107,6 +1108,10 @@ pub enum EventType {
     DisputeLost,
     MandateActive,
     MandateRevoked,
+    PayoutSuccess,
```

Comment:
Possible to add `PayoutCreated`? Trigger - when payout is created at connector's end.

Reply:
shouldn't we add feature flags here as well?